app/models/course.rb

```diff
@@ -126,6 +127,14 @@ def advance_start_at_duration=(time)
     settings(:course).advance_start_at_duration = time
   end
 
+  def braincert_whiteboard_api_key
```

Comment:
Since you already defined `app/models/course/lecture_settings.rb`, they key should be set there as well?